api/mod/modModel.js

```diff
@@ -78,6 +79,33 @@ const generateFaceoffs = () => {
   });
 };
 
+
+
+const generateVSequence = () =>{
+  return db.transaction(async (trx) =>{
+    try {
+      const data = await faceoff.getSubmissionsWithPoints(trx);
+      const foData = await ballot.getfaceOffData(trx);
+      let squads = ballot.groupBySquad(foData);
+      const childBallots = ballot.VSequence(squads, data);
+      for(let childNum in childBallots){
+          let votes = Object.assign(childBallots[childNum])
```

Comment:
To reduce the complexity of the code, I suggest that you declare `votes` outside of the for loop and update its value inside.